#! /bin/sh
patch -p1 -f $* < $0
exit $?

For the logo, see actually 01-w32-logo.patch.

2007-07-02  Marcus Brinkmann <mb@g10code.com>

	* w32-resource.rc: New file.
	* w32-logo.ico: New file.
	* src/Makefile.am: Define rules to compile resource files.
	(claws_mail_LDFLAGS): Add $(claws_mail_res_ldflag).
	(claws_mail_DEPENDENCIES): Add $(claws_mail_deps).
	(EXTRA_DIST): Add w32-logo.ico and w32-resource.rc.


diff -rupN claws-mail-2.9.2-p01/src/Makefile.am claws-mail-2.9.2/src/Makefile.am
--- claws-mail-2.9.2-p01/src/Makefile.am	2007-05-08 09:59:22.000000000 +0200
+++ claws-mail-2.9.2/src/Makefile.am	2007-07-02 19:45:47.000000000 +0200
@@ -14,6 +14,21 @@ install-exec-hook:
 	@rm -f $(DESTDIR)$(bindir)/sylpheed-claws
 	@ln -s claws-mail $(DESTDIR)$(bindir)/sylpheed-claws
 
+if PLATFORM_WIN32
+LTRCCOMPILE = $(LIBTOOL) --mode=compile $(RC) \
+     `echo $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) | \
+     sed -e 's/-I/--include-dir /g;s/-D/--define /g'`
+
+%.o : %.rc
+	$(LTRCCOMPILE) -i $< -o $@
+
+claws_mail_res_ldflag = -Wl,w32-resource.o
+claws_mail_deps = w32-resource.o
+else
+claws_mail_res_ldflag =
+claws_mail_deps =
+endif
+
 claws_mail_SOURCES = \
 	account.c \
 	action.c \
@@ -297,6 +312,7 @@ BUILT_SOURCES = \
 	quote_fmt_parse.h
 
 EXTRA_DIST = \
+	w32-logo.ico w32-resource.rc \
 	pixmaps/active.xpm \
 	pixmaps/addr_one.xpm \
 	pixmaps/addr_two.xpm \
@@ -454,7 +470,11 @@ arch_ldflags :=
 endif
 
 claws_mail_LDFLAGS = \
-	$(arch_ldflags) -export-dynamic
+	$(arch_ldflags) -export-dynamic $(claws_mail_res_ldflag)
+
+claws_mail_DEPENDENCIES = $(claws_mail_deps) \
+	$(etpan_library) \
+	gtk/libclawsgtk.la
 
 claws_mail_LDADD = \
 	$(etpan_library) \
diff -rupN claws-mail-2.9.2-p01/src/w32-resource.rc claws-mail-2.9.2/src/w32-resource.rc
--- claws-mail-2.9.2-p01/src/w32-resource.rc	1970-01-01 01:00:00.000000000 +0100
+++ claws-mail-2.9.2/src/w32-resource.rc	2007-07-02 19:46:07.000000000 +0200
@@ -0,0 +1 @@
+1 ICON "w32-logo.ico"
